_locales/en/messages.json

```diff
@@ -176,6 +194,15 @@
   "preview": {
     "message": "Preview"
   },
+  "addRow": {
+    "message": "Add table row"
```

Comment:
I'm not sure if we should use the word table in the UI for table settings - they don't actually look like tables, so I doubt users think of them as such. I suggest using "item" instead of "table row".